Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -105,4 +105,11 @@ theorem primeCounting'_add_le {a k : ℕ} (h0 : 0 < a) (h1 : a < k) (n : ℕ) :
 @[simp]
 theorem zeroth_prime_eq_two : nth Prime 0 = 2 := nth_count prime_two
 
+/-- The `n`th prime is greater or equal `n + 2`. -/
+lemma nth_prime_ge_add_two (n : ℕ): nth Prime n ≥ n + 2 := by
+  induction' n with n ih
+  · exact le_iff_eq_or_lt.mpr (Or.inl zeroth_prime_eq_two.symm)
+  · exact LE.le.trans (ge_iff_le.mp ((add_le_add_iff_right 1).mpr ih))
+      (succ_le_iff.mpr <| (nth_lt_nth infinite_setOf_prime).mpr <| lt_add_one n)
```

Comment:
```suggestion
/-- The `n`th prime is greater or equal `n + 2`. -/
lemma add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n := by
  induction' n with n ih
  · exact ge_of_eq zeroth_prime_eq_two
  · exact le_trans (add_le_add_right ih 1)
      (succ_le_of_lt <| nth_strictMono infinite_setOf_prime n.lt_succ_self)
```
`GT.gt`(`a ≥ b`) is defined as an abbreviation of `LT.lt` (`b ≤ a`) so using the latter is more preferable.
Also, we can golf (working piece of a proof as short as possible) here.